components/infobox/commons/infobox_basic.lua

```diff
@@ -31,4 +34,49 @@ function BasicInfobox:createBottomContent(infobox)
     return nil
 end
 
+--- Allows for using this for customCells
+function BasicInfobox:getMultiArgsForType(args, argType)
```

Comment:
"Type" is a bit of a problematic word since it can mean a lot of things, I would keep it simple:

```suggestion
function BasicInfobox:getAllArgs(args, base)
```

This should be nice and clear when you see it used:

```lua
self:getAllArgs(args, 'organizer')
```

A more specific version would be `getAllArgsForBase`